inc/health-check-curl-version.php

```diff
@@ -88,7 +88,7 @@ protected function get_curl_version() {
 	protected function is_recent_curl_version() {
 		$curl_version = $this->get_curl_version();
 
-		if ( $curl_version && version_compare( $curl_version, '7.20.0', '>=' ) ) {
+		if ( $curl_version && version_compare( $curl_version, '7.40.0', '>=' ) ) {
```

Comment:
7.4, not 7.40 :)

Reply:
@joost really not my area of expertise but after a quick search I see that:
- cURL 7.4 was released on October 16, 2000 https://curl.haxx.se/changes.html#7_4
- TLS 1.2 was defined in RFC 5246 in August 2008 https://en.wikipedia.org/wiki/Transport_Layer_Security#TLS_1.2

To my understanding the first cURL version that allowed to use TLS 1.2 is 7.34? https://curl.haxx.se/changes.html#7_34_0

Reply:
@jdevalk, not @ joost here :)

You're right. Let's go with 7.34 :)

Reply:
> @jdevalk, not @ joost here :)

I forgot 🤦‍♂